content/docs/tutorials/get-started/data-pipelines.md

```diff
@@ -0,0 +1,353 @@
+# Data Pipelines
+
+Support for **pipelines** is one of the key benefits of using DVC over simpler
+large file version control tools. _Data pipelines_ are well-defined series of
+data processing stages that produce some result.
+
+The pipeline in this tutorial explores a simple natural language processing
+(NLP) problem: to predict tags for a given Stack Overflow question. This is done
+in several stages, as shown in the figure below.
+
+![](/img/example-flow-2x.png) _Data modeling overview_
+
+> This is a simplified version of our [Deep Dive Tutorial](/doc/tutorials/deep).
+
+<details>
+
+### Expand to prepare the project
+
+If you just followed through the
+[versioning](/doc/tutorials/get-started/versioning-basics) page of this
+tutorial, you're all set. Otherwise, run these commands to get the project from
+Github:
+
+```dvc
+$ git clone https://github.com/iterative/example-get-started
+$ cd example-get-started
+$ git checkout 4-import-data
+$ dvc pull
+```
+
+</details>
+
+## Stages
+
+Data processing _stages_ are the nodes in the graph that forms our pipelines.
+They're computational units, for example machine learning scripts, that can take
+an input (dependency) and produce some result (output). Stage source code can be
+versioned with a regular Git workflow; We will then add a layer of metadata on
+top of it with DVC, to actually define each stage formally.
+
+### Connect code and data
+
+To go beyond versioning and sharing data, and achieve full reproducibility of
+our stages and pipelines, we'll need a way to connect code with the data it
+relates to.
+
+<details>
+
+### Expand to download example code
+
+Let's first get some code to work with:
+
+```dvc
+$ wget https://code.dvc.org/get-started/code.zip
+$ unzip code.zip
+$ rm -f code.zip
+```
```

Comment:
Should probably start using `tar` everywhere per iterative/example-repos-dev/issues/13

Reply:
Will be addressed for https://github.com/iterative/example-repos-dev/issues/13#issuecomment-614320329.